test/uk/gov/hmrc/merchandiseinbaggage/stubs/MibBackendStub.scala

```diff
@@ -89,7 +90,7 @@ object MibBackendStub extends MibConfiguration with CoreTestData {
   ): StubMapping =
     server
       .stubFor(
-        get(urlEqualTo(s"$declarationsUrl?mibReference=${mibReference.value}&eori=${eori.value}"))
+        get(urlEqualTo(s"${config.mibDeclarationsUrl}?mibReference=${mibReference.value}&eori=${eori.value}"))
```

Comment:
This can just be `get(urlEqualTo(s"/declare-commercial-goods/declarations?mibReference=${mibReference.value}&eori=${eori.value}"))`